### ISSUE TITLE
Unable to submit my app

### ISSUE THREAD

**Issue by vishal2647** [pinterest/PINRemoteImage#443]
Guys,
I am unable to submit my app on your website: https://developers.pinterest.com/apps/
I have already authorized my app and when i am trying to submit the app for review process, i am not able to submit it and i am getting “502 Bad Gateway” response and error message is: The MySQL server is running with the — read-only option so it cannot execute this statement.
This is not expected from reputed service like Pinterest. Please fix it as it is hampering our Business.

**Comment by garrettmoon** [comment]
@vishal2647 This isn't the proper place for this feedback.

**Comment by arunranga** [comment]
@vishal2647 : this has been fixed.

**Comment by vishal2647** [comment]
@arunranga  thanks a lot :)

**Comment by vishal2647** [comment]
@arunrangaI know this is not the right place to inform this but my app is in pending state from last 7 days. I tried reaching customer support in every possible way but no one is responding. I think writing about this issue here is my only chance. Please please help me.
My App ID: 4923335832214980114

**Comment by arunranga** [comment]
@vishal2647 : have a read of https://policy.pinterest.com/en/developer-guidelines, notably:

"Our developer API should only be used for direct-to-consumer integrations that help people discover and do what they love. And to avoid confusion with our Marketing Partners program, we don’t allow marketing platforms to use our developer API. This includes (but isn’t limited to) services that are meant to help marketers, advertisers or digital publishers (e.g. bloggers) report on, manage, distribute or optimize the digital marketing for paid or organic media.

In short, you shouldn’t use our developer API if your integration is intended to be authenticated by marketers, advertisers, businesses, or individuals with financial motivations for marketing purposes. You also can’t use our APIs to save, print or edit photos. If you are planning to use our API for one of these purposes or are unsure if this is relevant to you, let us know before building out your integration."

**Comment by vishal2647** [comment]
Thanks for response @arunranga  Please correct me if I am wrong- Our platform is similar to Buffer and Viraltag and they are using the same API. Just wondering how their application got approved?

**Comment by arunranga** [comment]
@vishal2647 : Buffer is part of the PMP program -- https://business.pinterest.com/en/pinterest-marketing-partners which uses non-public APIs.
(In general, comparing past apps that are approved is not helpful. And this is NOT the right forum for this.)

**Comment by vishal2647** [comment]
Ok understood.
We have revised the submission. Fingers crossed.
Thaks for all the information. Really appreciate it.

**Comment by vishal2647** [comment]
@arunranga It's been more than 20 days and my app still is in pending state. Sent 3 requests to customer support but no response so far. I don't know whom to contact next apart from writing about it here.

**Comment by tarekbadrshalaan** [comment]
I have the same issue and got 

`POST https://developers.pinterest.com/apps/{{app id}}/submission/ 502 (BAD GATEWAY)` 

any help !!

**Comment by thierryrene** [comment]
i have the same problem, and support don't return my email :|

**Comment by FerumFlex** [comment]
Same problem, more than one month my application is on review. Support says that you just need to wait.

**Comment by vishal2647** [comment]
Their developer program is dead....

**Comment by Adenion** [comment]
I have the same bug. 

> POST https://developers.pinterest.com/apps/{{app id}}/submission/ 502 (BAD GATEWAY) 

@arunranga please will you fix it ?

**Comment by spencerslickremix** [comment]
To all having submission APP troubles... after closer inspection of the 502 bad gateway error I can see an additional error in the console after clicking on it that that says..... 1406, u\"Data too long for column 'description' at row 1\"....

This error needs to absolutely display above the description box so users understand they are adding too much text to the area. An even better user experience here would to add some text that states how much text you can actually add.

**Comment by yosoychicho** [comment]
Same problem here. All attempts to create app are been ignored.

**Comment by justinslickremix** [comment]
Look at all these people having the same issue! @garrettmoon We realize this isn't the place to do this but we are trying to find solution here and no one at pinterest has responded to us in MONTHS. We develop one of the largest free social plugins that has over 80K active installs on WordPress websites. The amount of support we are getting on the daily because of our Pinterest APP not being approved is crazy. If you could put us in contact with someone who is going to help that would be greatly appreciated as we bring Pinterest a lot of traffic.

**Comment by YassineRjl** [comment]
Anyone managed to get this to work?

**Comment by realdonald1994** [comment]
It's still pending

**Comment by justinslickremix** [comment]
This is honestly the worst support we have received from any of the Social Networking companies' API we work with. Thankfully our plugins continue to grow without Pinterest as we now have over 90K active installs of our plugin on WordPress and Pinterest is losing out on millions of hits on their site from traffic our customers would be bringing to their site.
